torchvision/datasets/voc.py

```diff
@@ -0,0 +1,222 @@
+from __future__ import print_function
+import os
+import os.path
+import sys
+import torch
+import torch.utils.data as data
+from PIL import Image, ImageDraw
+if sys.version_info[0] == 2:
+    import xml.etree.cElementTree as ET
+else:
+    import xml.etree.ElementTree as ET
+
+VOC_CLASSES = ('background',  # always index 0
+               'aeroplane', 'bicycle', 'bird', 'boat',
+               'bottle', 'bus', 'car', 'cat', 'chair',
+               'cow', 'diningtable', 'dog', 'horse',
+               'motorbike', 'person', 'pottedplant',
+               'sheep', 'sofa', 'train', 'tvmonitor')
+
+# for making bounding boxes pretty
+COLORS = ((255, 0, 0, 128), (0, 255, 0, 128), (0, 0, 255, 128),
+          (0, 255, 255, 128), (255, 0, 255, 128), (255, 255, 0, 128))
+
+
+class VOCSegmentation(data.Dataset):
+    """VOC Segmentation Dataset Object
+    input and target are both images
+
+    NOTE: need to address https://github.com/pytorch/vision/issues/9
```

Comment:
Hi Francisco, any chance you've been able to make progress on the input/target random transforms?  I think those would be super useful.  Specifically, I have been toying with the idea of a random cropping transform that would crop both the image and resize target bounding boxes since currently that requires a bit of a work around without knowledge of whether the image was randomly cropped.